components/Wizard/CreateImageWizard.js

```diff
@@ -50,6 +51,15 @@ const CreateImageWizard = (props) => {
     setHasSaved(true);
   };
 
+  const intl = useIntl();
+
+  const messages = defineMessages({
+    createImage: {
+      id: "wizard.create.image",
+      defaultMessage: "Create image",
+    },
+  });
+
```

Comment:
We can define messages outside of the functional component. This reduces clutter as well as can prevent messages from rerendering especially in memoized components.